### PR TITLE
Update compatibility documentation and broken/outdated links

### PR DESCRIPTION
**Pull request by seut**
See commits.

### CODE REVIEW

File: docs/connect.rst [hunk 1]
```diff
@@ -96,10 +96,10 @@ Use the standard the `DBAL`_ or `Doctrine ORM`_ documentation for the rest of
 your setup process.
 
 .. _database user: https://crate.io/docs/crate/reference/en/latest/admin/user-management.html
-.. _DBAL documentation: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.7/index.html
-.. _DBAL: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.7/index.html
-.. _Doctrine provided example: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/configuration.html#obtaining-an-entitymanager
+.. _DBAL documentation: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.10/index.html
```
Comment (mkleen) [inline docs/connect.rst]: Is it worth referencing the same link twice once under `DBAL documentation` and `DBAL` ? Maybe on reference would be enough ?

File: docs/connect.rst [hunk 2]
```diff
@@ -96,10 +96,10 @@ Use the standard the `DBAL`_ or `Doctrine ORM`_ documentation for the rest of
...
+.. _DBAL: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.10/index.html
+.. _Doctrine provided example: https://www.doctrine-project.org/projects/doctrine-orm/en/2.7/reference/configuration.html#obtaining-an-entitymanager
 .. _Object-Relational Mapping: https://www.doctrine-project.org/projects/orm.html
-.. _ORM documentation: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/index.html
-.. _Doctrine ORM: https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/index.html
-.. _standard DBAL parameters: http://doctrine-orm.readthedocs.org/projects/doctrine-dbal/en/latest/reference/configuration.html
+.. _ORM documentation: https://www.doctrine-project.org/projects/doctrine-orm/en/2.7/index.html
```
Comment (mkleen) [inline docs/connect.rst]: Same here, twice the same link under 2 difference references.

File: docs/connect.rst (hunk 1 above)
Comment (seut) [reply docs/connect.rst]: They are now pointing correctly to 2 different targets.

Comment (mkleen) [reply docs/connect.rst]: Great, thank you!

**Pull request merged**